tests/test_price_parsing.py

```diff
@@ -80,7 +80,9 @@ def __eq__(self, other):
     Example(None, 'AED 8000 (USD 2179)',
             'AED', '8000', 8000),
     Example(None, '13800 ₶',
-            '₶', '13800', 13800)
+            '₶', '13800', 13800),
+    Example(None, '99¢',
```

Comment:
I think `99¢` should result in `Price(value=Decimal('0.99'), currency=None)`, since the cent symbol does not belong to a specific currency.

Reply:
This one is super-unclear, what to do exactly in such cases, as it may require changing the format. For example, $ also doesn't belong to a specific currency; we're not normalizing this to ISO codes at the moment. 

I'm not sure how cases like `1$ 99¢` are handled here - it seems they are not.

Reply:
They are not.
So how we can find what should be what?

Reply:
Given the current API, what about aiming for:
- `99¢` → `Price(value=Decimal('0.99'), currency=None)`
- `1$ 99¢` → `Price(value=Decimal('1.99'), currency='$')`

Reply:
`1$ 99¢` Is it a real example? Have we seen this before?

Reply:
That’s a great question. I assumed it was, but I’ve been searching for a few minutes and all I’ve been able to find are suggestions to use ¢ only up to 99, and never in combination with a regular currency symbol.

Reply:
Should we still aim for `99¢ → Price(value=Decimal('0.99'), currency=None)`? Or do you guys have reservations with that approach?

Reply:
@Gallaecio from what I can see, this would be the most correct interpretation.